lib/widgets/message_list.dart

```diff
@@ -342,7 +342,29 @@ class MessageListAppBarTitle extends StatelessWidget {
         final store = PerAccountStoreWidget.of(context);
         final stream = store.streams[streamId];
         final streamName = stream?.name ?? '(unknown channel)';
-        return _buildStreamRow(context, stream: stream, text: "$streamName > $topic");
+        // return _buildStreamRow(context, stream: stream, text: "$streamName > $topic");
+        return Row(
```

Comment:
`MessageListAppBarTitle` is probably not the right home for this. You can check out [`AppBar.actions`](https://api.flutter.dev/flutter/material/AppBar-class.html).

Reply:
yeah..seems fine to add it in the appbar actions..